about.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" type="text/css" href="./styles/mainStyle.css">
+  <link rel="stylesheet" type="text/css" href="./styles/mobileNav.css">
+  <link rel="stylesheet" type="text/css" href="./styles/aboutStyle.css">
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href="https://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav class="navigator-menu">
+      <img class="navigator-menu__burger" src="./images/icons/burgerIcon.png"> 
+    </nav>
+
+    <!-- nav menu -->
+
+    <div class="hidden-menu">
+      <ul class="hidden-menu__ul">
+        <li class="hidden-menu__li x-container"><img class="hidden-menu__x-element" src="./images/xIcon.png"></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option" href="./index.html">Home</a></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option">Buy</a></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option">Sell</a></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option">Sponsor</a></li>
+        <li class="hidden-menu__li"><a class="hidden-menu__option">News</a></li>
+        <li class="hidden-menu__li"><button type="submit" class="hidden-menu__button">Campain</button></li>
+      </ul>
+    </div>
+
+    <!-- Second nav bar -->
+
+    <div class="nav-bar-container">
+      <div class="nav-bar">
+        <div class="nav-bar__icons"><img src="./images/facebook.png"></div>
+        <div class="nav-bar__icons"><img src="./images/twitter.png"></div>
```

Comment:
- [ ] Kindly complete the first `nav` Header to have the `My page" and "Logout" Links. 
- For better user experience, these two are important for a standard webpage 

## Design given:
![image](https://github.com/EvansSnave/capstone-project/assets/50990925/01f2dea4-277e-407e-b435-3d566ec2ea3b)

## Your work:
![image](https://github.com/EvansSnave/capstone-project/assets/50990925/511efdba-7edb-400b-9bdb-c9746c6a0144)